modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/_mini_cart_item.scss

```diff
@@ -10,7 +10,7 @@
 		'info quantity price delete'
 		'errors errors errors errors';
 	grid-template-columns: auto 90px minmax(min-content, 100px) min-content;
-	overflow: hidden;
+	overflow: visible;
```

Comment:
ehi @dsitu what is the effect of this? does the scrollbar get visible? If yes I believe that maybe the right solution would be to work on the z-index of the popover.

Reply:
Hey @andrea-ale-sbarra, I tested this again and the scroll bar is **not** visible when the product name is longer. The product name gets cut off with "...". The popover shows up now and is shown over the cart items.